modules/xcode/xcode_common.lua

```diff
@@ -47,7 +47,14 @@
 		if node.isResource then
 			return "Resources"
 		end
-		return categories[path.getextension(node.name)]
+		local category = categories[path.getextension(node.name)]
+		if (not category)
+			and node.cfg
+			and (node.cfg.kind == p.SHAREDLIB or node.cfg.kind == p.STATICLIB)
```

Comment:
I don't think we split if statements over multiple lines like this anywhere else in the code base - this should probably just be on a single line for consistency.

Reply:
Sorry, I have a low vision and I tend to write code with small line length.

Reply:
I'm pretty sure I've used this style elsewhere in the past (though it may have been refactored away over time). I'm okay with the multiple lines in this case.